package.json

```diff
@@ -126,12 +126,15 @@
     "@istanbuljs/nyc-config-typescript": "1.0.2",
     "@nrwl/cli": "14.1.9",
     "@nrwl/cypress": "14.1.9",
+    "@nrwl/devkit": "14.1.9",
     "@nrwl/eslint-plugin-nx": "14.1.9",
     "@nrwl/jest": "14.1.9",
     "@nrwl/js": "14.1.9",
     "@nrwl/linter": "14.1.9",
     "@nrwl/node": "14.1.9",
     "@nrwl/nx-cloud": "latest",
+    "@nrwl/nx-plugin": "14.1.9",
+    "@nrwl/storybook": "14.1.9",
```

Comment:
It looks like the `apps/core-storybook/.storybook/main.ts` is importing from the root `.storybook/main.js` file. Is it possible to configure `.storybook/main.js` to be a TypeScript file?